Gibe.Umbraco.Blog/Utilities/ExactPhraseExamineValue.cs

```diff
@@ -12,7 +7,7 @@ public class ExactPhraseExamineValue : IExamineValue
 		public ExactPhraseExamineValue(string phrase)
 		{
 			Examineness = Examineness.Escaped;
-			Value = $"\"{phrase}\"";
+			Value = $"{phrase}";
```

Comment:
These were required for it to match the exact phrase... 

Are they not anymore? Are we sure?

Reply:
Oddly I could not yet it to work using this when I was testing out the author and tag filters. When I inspected the raw Lucene query and noticed the double quotes, I thought I'd try with just a single set, which did work.